.github/workflows/webviz-subsurface.yml

```diff
@@ -28,6 +28,15 @@ jobs:
 
     steps:
 
+    - name: Remove unused pre-installed software
+      run: |
+        # https://github.com/actions/virtual-environments/issues/751
+        # https://github.com/actions/virtual-environments/issues/709
+        sudo apt-get purge p7zip* yarn ruby-full zulu-*-azure-jdk* ghc* php7*
+        sudo apt-get autoremove
+        sudo apt-get clean
+        df -h
```

Comment:
Just to check, do you want to write disc usage each time to the log, or is it something you added to find folders you could delete?

Reply:
Intention to keep it - in order to easier see how much space is/was available at start of a  CI run.